server/routes/payment.js

```diff
@@ -37,35 +38,36 @@ router.get('/', (req, res) => {
     res.send("Welcome to payment route")
 })
 
-router.post('/payment', (req, res) => {
+router.post('/payment/:amount/:studentId', (req, res) => {
     let create_payment_json = {
         "intent": "sale",
         "payer": {
             "payment_method": "paypal"
         },
         "redirect_urls": {
-            "return_url": "http://localhost:5000/pay/success",
-            "cancel_url": "http://localhost:5000/pay/cancel"
+            //Redirect to next page
+            "return_url": `http://localhost:5000/pay/success/?studentid=${req.params.studentId}`,
```

Comment:
I'm not sure if this is the right url. It's fine for now, but add a TODO comment to replace it in the future! :)